backend/app/core/config.py

```diff
@@ -1,38 +1,23 @@
-from pydantic import BaseSettings
-
-
-class Settings(BaseSettings):
-
-    ENV = "dev"
-    database_url = "sqlite:///iW_dev.sqlite3"
-    title = "iWitness - Backend Development Application"
-    description = "A backend project with FastAPI for iWitness web application."
-    version = "0.0.0"
-    debug = True
-    docs_url = "/docs"
-    openapi_prefix = ""
-    openapi_url = "/openapi.json"
-    redoc_url = "/redoc"
-
-    @property
-    def fastapi_kwargs(self):
-        return {
-            "title": self.title,
-            "description": self.description,
-            "version": self.version,
-            "debug": self.debug,
-            "docs_url": self.docs_url,
-            "openapi_prefix": self.openapi_prefix,
-            "openapi_url": self.openapi_url,
-            "redoc_url": self.redoc_url,
-        }
-
-    def __str__(self):
-        return "Application Settings"
-
-
-def get_settings():
-
-    conf = Settings
-
+import logging
+from functools import lru_cache
+from typing import Dict, Type
+
+from app.core.settings.app_base_settings import AppBaseSettings, EnvTypes
+from app.core.settings.app_dev_settings import AppDevSettings
+from app.core.settings.app_prod_settings import AppProdSettings
+from app.core.settings.app_settings import AppSettings
+from app.core.settings.app_test_settings import AppTestSettings
+
+log = logging.getLogger("uvicorn")
+environments: Dict[EnvTypes, Type[AppSettings]] = {
```

Comment:
`environments` is the variable responsible to get us the correct settings. This is the fastest way to change settings (by choosing the value between `EnvTypes.PROD`,  `EnvTypes.DEV` , `EnvTypes.TEST` for `get_settings`)